ext/SerializationExt.jl

```diff
@@ -0,0 +1,83 @@
+module SerializationExt
+export serialize, deserialize
```

Comment:
I don't think this does anything (can't have exports from an extension, IIUC):
```suggestion
```

Reply:
Interesting, I needed these before to get some code I was using to access the proper serialization method, but for some reason that's not happening anymore.